__tests__/app.test.js

```diff
@@ -146,4 +146,14 @@ describe("/api/app", () => {
         expect(body.msg).toBe("No reviews found for review_id: 10000");
       });
   });
+  test("Post a new entry into the comment", () => {
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send({ author: "mallionaire", body: "hi i'm mallionaire" })
```

Comment:
This should be username rather than author in the request body. The response should have this as author, though